### PR TITLE
SE-702: Moving valuations, adding inline recipe tests

### PR DESCRIPTION
**Pull request by cscialino**
# Pull Request Checklist

- [ x ] Read the contributing guidelines
- [ x ] Tests pass
- [ x ] Raised the PR against the develop branch

# Description of the PR

- Moving valuations to new directory Valuations within tests/tutorials , other valuation/recipe tests will be added here.
- Extended valuation example from use of in-line recipe to use of upserted recipe

### CODE REVIEW

File: sdk/tests/tutorials/valuation/test_valuation.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+import lusid
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class Valuation(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```
Comment (rizwansaeed) [inline sdk/tests/tutorials/valuation/test_valuation.py:15]: can we add a corresponding `tearDown` to delete the portfolio after we're done

File: sdk/tests/tutorials/valuation/test_valuation.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+            (cls.instrument_ids[0], 100),
+            (cls.instrument_ids[1], 200),
+            (cls.instrument_ids[2], 300),
+        ]
+
+        requests = [
+            models.UpsertQuoteRequest(
+                quote_id=models.QuoteId(
+                    models.QuoteSeriesId(
+                        provider="DataScope",
```
Comment (rizwansaeed) [inline sdk/tests/tutorials/valuation/test_valuation.py:92]: this should be `Lusid`, `DataScope` is a specific Refinitiv product

File: sdk/tests/tutorials/valuation/test_valuation.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        :param str recipe_code: The code of the the configuration recipe
+        :return: ConfigurationRecipe
+        """
+
+        return models.ConfigurationRecipe(
+            scope=recipe_scope,
+            code=recipe_code,
+            market=models.MarketContext(
+                market_rules=[],
+                suppliers=models.MarketContextSuppliers(equity="DataScope"),
```
Comment (rizwansaeed) [inline sdk/tests/tutorials/valuation/test_valuation.py:128]: change `DataScope` > `Lusid`

File: sdk/tests/tutorials/valuation/test_valuation.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        )
+
+        # Complete aggregation
+        aggregation = self.aggregation_api.get_aggregation(
+            scope=TestDataUtilities.tutorials_scope,
+            code=self.portfolio_code,
+            aggregation_request=aggregation_request,
+        )
+
+        for item in aggregation.data:
```
Comment (rizwansaeed) [inline sdk/tests/tutorials/valuation/test_valuation.py:213]: do we need this?

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

File: sdk/tests/tutorials/valuation/test_valuation.py [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+            ],
+            [
+                "Test valuation with an aggregation request using an already upserted recipe",
+                None,
+                "TestRecipes",
+                "SimpleQuotes",
+            ],
+        ]
+    )
+    def test_aggregation(self, _, in_line_recipe, recipe_scope, recipe_code) -> None:
```
Comment (rizwansaeed) [inline sdk/tests/tutorials/valuation/test_valuation.py:203]: need to add in https://github.com/finbourne/lusid-features-python too, which may need adding a new one to https://gitlab.finbourne.com/salesengineering/lusid-features/-/blob/master/features.yml

Comment (cscialino) [reply sdk/tests/tutorials/valuation/test_valuation.py:203]: I think there is an issue currently with @lusidfeature decorators, @slacki123 is looking into a fix. I will update it as a comment for now, which I can then amend and also update a feature code to each example + yaml file.

Comment (cscialino) [reply sdk/tests/tutorials/valuation/test_valuation.py:204]: Actually it works in this instance, so I've added it. Having a look at lusidfeature as well. @rizwansaeed

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**